examples/todoapp/lib/feature_todo_list/di/todo_list_page_with_dependencies.dart

```diff
@@ -23,5 +24,11 @@ class TodoListPageWithDependencies extends StatelessWidget {
             context.read(),
           ),
         ),
+        RxBlocProvider<TodoListBulkEditBlocType>(
```

Comment:
Don't we register this bloc at the global DI ?

Reply:
I will move it there, but it wasn't registered there.

Reply:
As discussed the `TodoListBulkEditBlocType` must be registered in its own lib